src/main.rs

```diff
@@ -841,13 +841,9 @@ fn real_main() -> Result<(), VetError> {
 
     info!("Running: {:#?}", cmd.cargo_command());
 
-    let metadata = match cmd.exec() {
-        Ok(metadata) => metadata,
-        Err(e) => {
-            error!("'cargo metadata' failed: {}", e);
-            panic_any(ExitPanic(-1));
-        }
-    };
+    let metadata = cmd
+        .exec()
+        .wrap_err("'cargo metadata' exited unsuccessfully")?;
```

Comment:
I noticed that you concatenated errors manually in a lot of places. Since eyre already does that automatically for source errors I opted to replace these with calls to `wrap_err` which creates a wrapping error from the given string and sets the original error as the source of that error.